### PR TITLE
Added styles for Docs section, improved docs pages appearance

### PR DESCRIPTION
**Pull request by Y-Shcheholskyy**
Added styles for Docs section

- Added styles for docs page;
- Added some navigation between docs page and docs chapters;

Improved docs pages appearance

- Added indents between titles and other docs parts to provide right consistency;
- Fixed indents at the edges of the docs pages;
- Aligned "copy" buttons for code snippets

### CODE REVIEW

**Comment by bhoran** [comment]
There's jumping on the main text (I think this is if the bullets in the left nav break over multiple lines)
https://github.com/graalvm/graalpy.github.io/assets/12658122/55786b0f-3c1b-48f3-8082-ec9929b4c063

**Comment by bhoran** [comment]
The next/previous chapter isn't appropriate for the reference content.

**Comment by bhoran** [comment]
The indentation has gone wrong
Screenshot 2024-01-16 at 10 37 26
(The <ol><li> elements are too far left)

File: docs/graalpy-embedded/reference/Interoperability.md [hunk 1]
```diff
@@ -25,7 +34,7 @@ GraalVM uses the Python API to run Python native extensions with the [GraalVM LL
 1. Install other languages into GraalPy. 
 For that, use `libexec/graalpy-polyglot-get` from the distribution's root directory.
 To install Ruby, for example:
-    ```shell
+    ```python
```
Comment (bhoran) [inline docs/graalpy-embedded/reference/Interoperability.md:37]:
```suggestion
    ```shell
```

File: docs/graalpy-embedded/reference/Interoperability.md [hunk 2]
```diff
@@ -157,4 +166,29 @@ Types not listed in the below table below currently have no special interpretati
 | `exception` | Any Python `BaseException` subtype. |
 | `MetaObject`   | Any Python `type`. |
 | `executable` | Any Python object  with a `__call__` method. |
-| `instantiable`` | Any Python `type`. |
\ No newline at end of file
+| `instantiable`` | Any Python `type`. |
```
Comment (bhoran) [inline docs/graalpy-embedded/reference/Interoperability.md:169]:
```suggestion
| `instantiable` | Any Python `type`. |
```

File: docs/graalpy-embedded/reference/Interoperability.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ permalink: /docs/polyglot-api/
 toc: true
 ---
 
+<div class="back-nav">
```
Comment (timfel) [inline docs/graalpy-embedded/reference/Interoperability.md:8]: Repetetive and not page-specific content like this should not be in these files, it should be in the layout, potentially parameterized.

File: _layouts/documentation.html [hunk 4]
```diff
@@ -2,4 +2,82 @@
...
+        </div>
+    </div>
+</section>
+
+<section class="edu__body">
+    <div class="graal-docs">
+        <div class="graal-docs__row">
+            <div class="graal-docs__column">
+                <div class="graal-docs__card">
+                    <img src='{{ "/resources/img/docs-page/miscellaneous-book.svg" | relative_url }}'
```
Comment (timfel) [inline _layouts/documentation.html:21]: This is not how layouts work - the layout should have non-page specific content, all of these strings should be defined in the Reference.md and passed into the layout via site variables.

Comment (timfel) [reply _layouts/documentation.html:21]: The examples layout pages are wrong in the same way, and I fixed that in https://github.com/graalvm/graalpy.github.io/pull/34

File: docs/graalpy-embedded/reference/Interoperability.md (hunk 3 above)
Comment (Y-Shcheholskyy) [reply docs/graalpy-embedded/reference/Interoperability.md:8]: Got it, I'll take it into account for the future

File: _layouts/documentation.html (hunk 4 above)
Comment (Y-Shcheholskyy) [reply _layouts/documentation.html:21]: I see, thanks

**Comment by Y-Shcheholskyy** [comment]
> I see what you're trying to do with the navigation @Y-Shcheholskyy, but it's not the way it should be done with Jekyll. Do you mind if I update this PR?

Sure, Bernard and I talked about it today. We agreed that this navigation not suit us so I planned to rewrite it anyway.

**Comment by Y-Shcheholskyy** [comment]
There have been many changes, including in references, so I will close this PR and open a new one

**Pull request closed without merging**